recipe/meta.yaml

```diff
@@ -1,5 +1,5 @@
 {% set name = "f2c" %}
-{% set version = "20210323183018" %}
+{% set version = "20200916" %}
```

Comment:
It's the same, see https://github.com/AnacondaRecipes/f2c-feedstock/pull/2#issuecomment-1555339252 and https://github.com/AnacondaRecipes/f2c-feedstock/pull/2#issuecomment-1555349654. I verified the content of both and they 100% match, bit for bit.

Reply:
Yes, it isn't truly a "new 2021" version, it's a misleading name.

Reply:
I see, thanks for clarification

Reply:
yes, I see. thanks for clarifying this to me.  So the only thing I would love to see then is that the patches are removed from repo. Otherwise I am fine